### ISSUE TITLE
battery holder

### ISSUE THREAD

**Issue by mach0** [Vault2501/Geiger-Counter-Cajoe#1]
I cannot find any information about the battery/used battery holder.
Can you please provide which type this is? Is the battery being charged when there is USB power connected?

**Comment by mach0** [comment]
think I found it .. one picture showed (after zooming in) a 18650 battery and I assume the "holder" is this one:
https://www.az-delivery.de/en/products/battery-expansion-shield-18650-v3-inkl-usb-kabel

**Comment by Vault2501** [comment]
The one I got is slightly different, it has a button instead of a switch, and on the side of the button, there also is a usb-c charging port. That's why I face this side to the case wall and have openings for it. It's switched on and off by pushing the button. Also, the battery is charging while it provides power to the ESP, but when you plug in the usb-c plug to charge it, it switches off for a brief moment, rebooting the controller. Which in my case is no issue, but for uninterrupted power, you need an additional circuit.
The one I got looks like this one (I had them laying around and am not sure anymore where I got them): https://www.amazon.de/Heemol-Battery-Shield-Anschluss-Ausgang/dp/B0B6GCGYDC?th=1

**Comment by mach0** [comment]
Sorry to bother you again,
I found such battery holders but mine are more or less mirrored.
So I need to move the bat to the other side and adjust the on/off switch and the power hole.
Also I have an oled board with 1.3".

Trying to find the correct "switches" in the scad file I found:
bat_con_usbc_off_x=18.5 (which I set to  +81 which seems to be the correct value for that
bat_con_button_off_x=85.3 (which i set to +8  which also seems pretty correct for the opening)
for the oled_board i set:
oled_display_x=35
oled_display_y=19
which makes a nice oled opening, but the "mounting squares" at the bottom are wrong now

and also I am setting this:
translate([wall+gap+8.8,case_y-gap-bat_y-wall,wall]) // 8.8 to move it to the other X side
that works very well for the bottom (but I am not sure if this is the correct place to make that change) and I also cannot find the place where I can move the top-case screws.

to sum it up: Can you please help me with:

1: where can I adjust the OLED mounting tips (and probably inverse the because I do not have squares but rather zylinders to mount the display (https://www.az-delivery.de/products/1-3zoll-i2c-oled-display)

2: where can I move the top-case screws to the other side? (probably mirror them in the middle?

I am a very beginner with scad, but it does look very nice - if only I could fully understand your code.

from the current view I would say the bottom is finished (though I do not know it i did it correctly)
If you want to have a look I can fork your repo and make a pull request - or you could also take a look at my repository than.
